.github/workflows/examples_echo.yaml

```diff
@@ -0,0 +1,23 @@
+name: examples/echo
```

Comment:
Didn't notice it before, but dont we always reflect the name to the file name of the workflow?

Reply:
I did it in an attempt to make it clear the check corresponds to the examples/echo directory